header/p2p/peer_tracker.go

```diff
@@ -71,8 +71,8 @@ func (p *peerTracker) track() {
 	}()
 
 	// store peers that have been already connected
-	for _, peer := range p.host.Peerstore().Peers() {
-		p.connected(peer)
```

Comment:
* We have in memory peerstore, which is reset on which restart. This was only useful to add trusted peers, preset there during header module construction. Otherwise, it's useless for us.
* Also, we need this change because the logic of  `p.connected` assumes it's called only once for a peer. If we have trusted peers in the peer store and then connect, it will execute connected twice on the same peer.